net-im/toxic/toxic-0.8.2.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+
+inherit python-single-r1 xdg
+
+DESCRIPTION="A curses-based client for Tox"
+HOMEPAGE="https://github.com/JFreegman/toxic"
+PATCHES_GIST="35e877418de56afd7305dd55c89b536e"
+PATCHES_REV="fe985af45f1afe73f1e79bc1ffb03c6ac2d25c88"
+SRC_URI="https://github.com/JFreegman/toxic/archive/v${PV}.tar.gz -> ${P}.tar.gz
+		 https://gist.github.com/fx-carton/${PATCHES_GIST}/archive/${PATCHES_REV}.zip -> ${P}_patches.zip"
```

Comment:
Use one tab to indent. And please, please don't use .zip unless absolutely inevitable. You'd have to add dependency on unzip and nobody likes that, so just use `.tar*`.